### DISCUSSION TITLE
How to interpret Excavator github actions logs?

### DISCUSSION THREAD

**Discussion by Slach** [ScoopInstaller/Extras#11353] [Q&A] [answered]
on https://github.com/BloopAI/bloop/releases/tag/v0.4.1 i see new version of bloop released 

but on 
https://github.com/ScoopInstaller/Extras/actions/runs/5096007043/jobs/9161486481#step:3:421
i see
```
bloop: 0.4.1 (scoop version is 0.3.4) autoupdate available
Autoupdating bloop
Downloading bloop_0.4.1_x64_en-US.msi to compute hashes!
VERBOSE: GET with 0-byte payload
VERBOSE: received -byte response of content type application/json
ERROR Response status code does not indicate success: 403 (Forbidden).
aws-nosql-workbench: couldn't match 'NoSQL Workbench-win-([\d.]+)\.exe' in https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/workbench.settingup.html
```

What exactly it means is 403 error related to bloop msi or to aws-nosql-workbench?  
If yes
then why 403 return to following url
https://github.com/BloopAI/bloop/releases/download/v0.4.1/bloop_0.4.1_x64_en-US.msi which have the same format as previous release ?

**Comment by StarsbySea** [comment, marked as answer]
Github has a rate limit on accesses from Github Action, so sometimes this will cause the download of the file to fail (403), usually waiting for the next automatic check for updates will do the trick.

If this solves your problem, please consider marking it as answered.
